webapp/src/components/CollectionItem.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import CreateIcon from "./CreateIcon";
+import {
+  ICON_TYPES_LIST,
+  INNER_ICON_TYPES_LIST,
+} from "../constants/iconTypeConsts";
+
+export default function CollectionItem({linkName, url}) {
+  return (
+    <div className="collection-item d-flex align-items-center justify-content-evenly px-1">
+      <div className="d-flex align-items-center justify-content-evenly">
+        <div>
+          <CreateIcon
+            bgColour={"#ffffff"}
+            customStyles={{
+              width: "25px",
+              height: "25px",
+              borderRadius: "50%",
+              border: "2px solid #ffffff",
+              boxShadow: "0px 17.7395px 125px rgba(48, 56, 125, 0.32)",
+            }}
+            innerIconBg={"/images/link.png"}
+            innerIconCustomStyles={{
+              width: "22px",
+              height: "22px",
+              backgroundRepeat: "no-repeat",
+              backgroundPosition: "center",
+              backgroundSize: "cover",
+            }}
+          />
+        </div>
+        <div className="mx-3">
+          <p className="para1 my-auto ">Link name goes here</p>
+        </div>
+      </div>
+      <div
+        className=""
+        style={{
+          width: "400px",
```

Comment:
Convert inline styling to class-based. Check and make changes for all the usages within this file.